backend/inspirehep/mailing/errors.py

```diff
@@ -8,7 +8,3 @@
 
 class MailingError(Exception):
```

Comment:
It's not used anywhere now. Please remove it.

Reply:
yeah, this code is keeping changing and I am constantly rebasing.